backend/overstroomik_service/rijksdriehoek.py

```diff
@@ -0,0 +1,98 @@
+"""
+Copyright Jonathan de Bruin (2016)
```

Comment:
Well, not sure if this is allowed, what kind of license was this code? Also, we might not need it at all.

Reply:
I don't need anymore, removed from code